tests/test_contrib/test_scatter.py

```diff
@@ -113,7 +113,7 @@ def test_scatter_xy_changes_to_features(self):
         Assert that x,y with no features will not raise scatterviz error
         """
         visualizer = ScatterViz(x='one', y='two')
-        assert visualizer.features_ == ['one', 'two']
+        assert visualizer.features == ['one', 'two']
```

Comment:
This test ensures that the "old functionality" is maintained; see the note in `scatter.py` for more detail.